.nuspec/Microsoft.Maui.Controls.props

```diff
@@ -6,6 +6,8 @@
 		<_IsHotRestartDefined>$([System.Text.RegularExpressions.Regex]::IsMatch('$(DefineConstants.Trim())', '(^|;)HOTRESTART($|;)'))</_IsHotRestartDefined>
 		<DefineConstants Condition="!$(_IsHotRestartDefined) And '$(IsHotRestartBuild)' == 'true'">HOTRESTART;$(DefineConstants)</DefineConstants>
 		<UseInterpreter Condition="'$(UseInterpreter)' == '' and '$(Configuration)' == 'Debug'">True</UseInterpreter>
+		<MtouchLink Condition="'$(MtouchLink)' == '' and '$(Configuration)' == 'Debug' and '$(UseInterpreter)' == 'true'">None</MtouchLink>
+		<LinkMode Condition="'$(LinkMode)' == '' and '$(Configuration)' == 'Debug' and '$(UseInterpreter)' == 'true'">None</LinkMode>
```

Comment:
Why is this not in the `ios`, `maccatalyst` workload instead?

You can also name configurations other than `Debug`, which is why we have a `$(_AndroidIncludeDebugSymbols)` based on `$(DebugType)` and `$(DebugSymbols)` for checks like this.

Reply:
Probably so the current maui can work in the current release. But it needs to move into ios and then we can delete.

Reply:
At this point, the call is whether or not it makes the bar for SR2. If not, then it moves to ios repo.

Reply:
Move to iOS repo, doesnt fixe anything critical for our users.

Reply:
@rmarinho It fixes C# and XAML Hot Reload working for users running iOS devices and changing this makes the default experience much better than launching and throwing an error bar. 

I don't care if it lives here, or in the workload, or whatever. I just need to know when it will land so we can tell people when it will be working out of the box.